phpBB/styles/prosilver/theme/content.css

```diff
@@ -491,18 +491,17 @@ blockquote cite > div {
 
 /* Code block */
 .codebox {
-	padding: 3px;
 	border: 1px solid transparent;
 	font-size: 1em;
 	margin: 1em 0 1.2em 0;
-	overflow-x: scroll;
 	word-wrap: normal;
 }
 
 .codebox p {
 	text-transform: uppercase;
 	border-bottom: 1px solid transparent;
-	margin-bottom: 3px;
+	margin-bottom: 0;
+	padding: 3px 3px 0 3px;
```

Comment:
The problem is that I have to remove `.codebox {overflow-x: scroll;}` to get rid of the double scrollbar and keep the overflow for the inner `<code>`, because having an overflow property on the parent would make the "Code: Select all" to scroll as well).

Doing just that fixes the problem, but the scrollbars have margins:

![scrollbar-margin](https://cloud.githubusercontent.com/assets/7398634/25627919/f234f176-2f6c-11e7-968b-b0f866db00cb.png)

Do we want margins?

However, after the padding and margin changes, it looks like this:

![scrollbar-nomargin](https://cloud.githubusercontent.com/assets/7398634/25627949/0de4f402-2f6d-11e7-94c9-77b98f1e309d.png)